addon/components/nrg-list/header/search/component.js

```diff
@@ -43,7 +43,7 @@ export default Component.extend(GlobalKeyboardShortcutsMixin, {
 
   actions: {
     focus() {
-      this.$('input').focus();
+      this.$('input')[0].focus();
```

Comment:
Are you sure this fixes your issue? `this.$('input')` should only ever get one `input` since it is in context of the component.